barpyrus/widgets.py

```diff
@@ -44,7 +44,11 @@ def next_timeout(self):
     def maybe_timeout(self, now):
         some_timeout = False
         for w in self.subwidgets:
-            some_timeout = w.maybe_timeout(now) or some_timeout
+            try:
+                some_timeout = w.maybe_timeout(now) or some_timeout
+            except:
+                breakpoint()
+                raise RuntimeError()
```

Comment:
I think this was just for you and does not belong to this PR, doesn't it?

Reply:
Woops, indeed